detox/test/src/app.js

```diff
@@ -149,7 +149,11 @@ export default class example extends Component {
 
   renderInnerScreen() {
     const Screen = this.state.screen;
-    return <Screen setScreen={this.setScreen}/>;
+    return (
+      <View style={{flex: 1, paddingTop: 24, paddingBottom: 50}}>
```

Comment:
I don't like this direction... why can't we just listen to the insets natively and apply them?